apps/desktop-wallet/src/utils/app-data.ts

```diff
@@ -21,15 +21,20 @@ export const KEY_APPMETADATA = 'alephium/desktop-wallet/appmetadata'
 export const currentVersion: string = import.meta.env.VITE_VERSION
 export const isRcVersion: boolean = currentVersion.includes('-rc.')
 
-export interface AppMetaData {
-  lastTimeGitHubApiWasCalled: Date
+export interface AppMetadataGitHub {
+  lastTimeGitHubApiWasCalledForLatestVersion: Date
```

Comment:
`lastGithubCallForLatestVersion: Date` ?

Reply:
Don't you think we should be as clear as possible to avoid the readability problems we faced recently?